libs/common/src/mongo-db-client/index.ts

```diff
@@ -1,92 +1,122 @@
 import { Collection, Db as DbConnection, MongoClient, WithoutId } from 'mongodb';
+import dotenv from 'dotenv';
+import { ClientConnectionStatus } from '../types';
 import { MongoDbCollectionName } from '../types/mongo-db-models/mongo-db-collection-name';
 import { DbModel } from '../types/mongo-db-models/db-model';
 
-type ConnectionOptions = {
-  dbConnectionString: string;
-  dbName: string;
-};
+dotenv.config();
 
-type MongoDbClientConnection =
-  | {
-      isInitialised: false;
-    }
-  | {
-      isInitialised: true;
-      client: MongoClient;
-      connection: DbConnection;
-    };
+const { MONGODB_URI, MONGO_INITDB_DATABASE } = process.env;
 
+/**
+ * Class representing a MongoDB client.
+ */
 export class MongoDbClient {
-  private dbName: string;
+  /**
+   * MongoDB connection string.
+   */
+  private uri: string = MONGODB_URI as string;
```

Comment:
out of curiosity what is MONGODB_URI typed atm? is it any or something instead of string?

Reply:
There are some Zod schema validators hanging around which you can pass your environment into to extract values and also check they are for example strings (and then it types them to string) which might be quite helpful for this file

Reply:
Sorry I think I mean more right now we have uri: string = MONGODB_URI and we are casting MONGODB_URI to string,w as wondering what the compiler thinks MONGODB_URI is. It's minor so happy to resolve anyway